contracts.md

```diff
@@ -1,45 +1,79 @@
-* DelegationManager -- implementation	0x1784be6401339fc0fedf7e9379409f5c1bfe9dda
-* DelegationManager -- Transparent Upgradeable Proxy	0x39053D51B77DC0d36036Fc1fCc8Cb819df8Ef37A
-* StrategyManager -- implementation	0x70f44c13944d49a236e3cd7a94f48f5dab6c619b
-* StrategyManager -- Transparent Upgradeable Proxy	0x858646372CC42E1A627fcE94aa7A7033e7CF075A
-* RewardsCoordinator -- implementation	0x5bf7c13D5FAdba224ECB3D5C0a67A231D1628785
-* RewardsCoordinator -- Transparent Upgradeable Proxy	0x7750d328b314EfFa365A0402CcfD489B80B0adda
-	
-### EIGENLAYER -- EIGENPODS CONTRACTS	
-* EigenPod -- implementation	0x28144c53ba98b4e909df5bc7ca33eaf0404cffcc
-* EigenPodBeacon (UpgradeableBeacon contract)	0x5a2a4F2F3C18f09179B6703e63D9eDD165909073
-* EigenPodManager -- implementation	0xe4297e3dadbc7d99e26a2954820f514cb50c5762
-* EigenPodManager -- Transparent Upgradeable Proxy	0x91E677b07F7AF907ec9a428aafA9fc14a0d3A338
-* DelayedWithdrawalRouter -- implementation	0x4bb6731b02314d40abbffbc4540f508874014226
-* DelayedWithdrawalRouter -- Transparent Upgradeable Proxy	0x7Fe7E9CC0F274d2435AD5d56D5fa73E47F6A23D8
-* AVSDirectory -- implementation	0xdabdb3cd346b7d5f5779b0b614ede1cc9dcba5b7
-* AVSDirectory -- Transparent Upgradeable Proxy	0x135dda560e946695d6f155dacafc6f1f25c1f5af
-	
-### EIGENLAYER -- STRATEGY CONTRACTS	
-* StrategyBaseTVLLimits -- implementation	0xdfdA04f980bE6A64E3607c95Ca26012Ab9aA46d3
-* StrategyBaseTVLLimits -- TUP -- stETH	0x93c4b944D05dfe6df7645A86cd2206016c51564D
-* StrategyBaseTVLLimits -- TUP -- rETH	0x1BeE69b7dFFfA4E2d53C2a2Df135C388AD25dCD2
-* StrategyBaseTVLLimits -- TUP -- cbETH	0x54945180dB7943c0ed0FEE7EdaB2Bd24620256bc
-* StrategyBaseTVLLimits -- TUP -- ETHx	0x9d7eD45EE2E8FC5482fa2428f15C971e6369011d
-* StrategyBaseTVLLimits -- TUP -- ankrETH	0x13760F50a9d7377e4F20CB8CF9e4c26586c658ff
-* StrategyBaseTVLLimits -- TUP -- oETH	0xa4C637e0F704745D182e4D38cAb7E7485321d059
-* StrategyBaseTVLLimits -- TUP -- osETH	0x57ba429517c3473B6d34CA9aCd56c0e735b94c02
-* StrategyBaseTVLLimits -- TUP -- swETH	0x0Fe4F44beE93503346A3Ac9EE5A26b130a5796d6
-* StrategyBaseTVLLimits -- TUP -- wBETH	0x7CA911E83dabf90C90dD3De5411a10F1A6112184
-* StrategyBaseTVLLimits -- TUP -- sfrxETH	0x8CA7A5d6f3acd3A7A8bC468a8CD0FB14B6BD28b6
-* StrategyBaseTVLLimits -- TUP -- lsETH	0xAe60d8180437b5C34bB956822ac2710972584473
-* StrategyBaseTVLLimits -- TUP -- mETH	0x298aFB19A105D59E74658C4C334Ff360BadE6dd2
-* EigenStrategy -- Implementation	0x27e7a3a81741b9fcc5ad7edcbf9f8a72a5c00428
-* EigenStrategy -- TUP - EIGEN	0xaCB55C530Acdb2849e6d4f36992Cd8c9D50ED8F7
-
-### EIGEN & bEIGEN – Token Contracts
-* EIGEN – 0xec53bf9167f50cdeb3ae105f56099aaab9061f83
-* bEIGEN – 0x83E9115d334D248Ce39a6f36144aEaB5b3456e75
-
-*subject to change and additions*
+This table attempts to capture all smart contracts that are important to the functioning of the EigenLayer protocol. It is subject to change and additions, particularly as the result of ELIPs.
```

Comment:
Should this be something more relevant to this repository like:

"This table captures all smart contracts included in the EigenLayer protocol that are governed by the ELIP process."

Or does that not make sense?